validateJsonSchema.js

```diff
@@ -9,7 +9,8 @@ const jsonSchemaFolders = [
     "kontrakter/politi/varetekt",
     "kontrakter/politi/fengsling",
     "kontrakter/da/varetekt",
-    "kontrakter/felles/kvittering"
+    "kontrakter/felles/kvittering",
+    "kontrakter/felles/kodeverk"
```

Comment:
Bra å validere kodeverket etter schema, blir bare litt rart at selve kodeverket må ligge under mappe `eksempelfiler` as of oppsettet nå? Kan jo diskutere en annen løsning som blir litt mer ryddig

Reply:
ekesmpelfiler inneholder bare et eksempel, faktisk kodeverk definisjon ligger under folderen ESAS/kodeverk/
Det er lagt til validering av også disse filene.